### PR TITLE
release/1.10.0

### PR DESCRIPTION
**Pull request by vmalaviya-splunk**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
 # Splunk Enterprise SDK for JavaScript Changelog
 
+## v1.10.0
+
+### Minor changes
+
+* Added support for Splunk Enterprise 8.2
+* Added support for Node v14.
+* Dropped support for deprecated request http client.
+* Added support for [needle](https://www.npmjs.com/package/needle) http client library.
```
Comment (fantavlik) [inline CHANGELOG.md:10]: Actually I think these two warrant moving into a new ```### Major changes``` section even though they are not breaking changes:
```
### Major changes
* Dropped support for deprecated request http client.
* Added support for [needle](https://www.npmjs.com/package/needle) http client library.
* ```

**Review by fantavlik — APPROVED** [review]
Looks good, thanks for the thorough CHANGELOG.md writeup - can you just make the minor change to add the requests -> needle items to a `Major Changes` section and then feel free to merge this?

**Comment by vmalaviya-splunk** [comment]
Consider it done.

File: CHANGELOG.md (hunk 1 above)
Comment (vmalaviya-splunk) [reply CHANGELOG.md]: Changes done

**Pull request merged**